sources/packages/forms/src/form-definitions/sfaa2021-22.json

```diff
@@ -1574,7 +1565,7 @@
               "validateOn": "change",
               "validate": {
                 "required": true,
-                "onlyAvailableItems": true,
+                "onlyAvailableItems": false,
```

Comment:
Having this as false demands that the API validate the values otherwise the user can input any value here.
Please ensure that the API is validating `Full Time`/`Part Time` based on the token configuration.

Reply:
Was this verification added?

Reply:
Added it now. Apologies, missed it earlier.